lib/gyro/parser/xcdatamodel/xcdatamodel.rb

```diff
@@ -37,18 +37,20 @@ class XCDataModel
 
         def initialize(xcdatamodel_dir)
           is_xcdatamodeld = xcdatamodel_dir.extname != '.xcdatamodeld'
-          Gyro::Log.fail!('Please target an xcdatamodel inside your xcdatamodeld') unless is_xcdatamodeld
+          Gyro::Log.fail!('Please target an \'.xcdatamodel\' file inside your xcdatamodeld') unless is_xcdatamodeld
           if xcdatamodel_dir.parent.extname == '.xcdatamodeld'
             xcdatamodeld_info_message = 'You are using an xcdatamodeld, ' \
                                         'please be sure you target the correct version of your xcdatamodel.' \
-                                        ' Actual version using by gyro is : ' + xcdatamodel_dir.basename.to_path
+                                        ' Current version used by gyro is: ' + xcdatamodel_dir.basename.to_path
             Gyro::Log.info(xcdatamodeld_info_message)
           end
           file_contents = xcdatamodel_dir + 'contents'
           Gyro::Log.fail!('Unable to find contents of xcdatamodel') unless file_contents.exist?
           @entities = {}
-          xml_document = Document.new(file_contents.open)
-          load_entities(xml_document)
+          file = File.open(file_contents)
+          document_xml = Document.new(file)
+          file.close
```

Comment:
`File.open` can take a trailing closure/block which will open the file, execute the block with the file handle, then auto-close the file.

```ruby
document_xml = File.open(file_contents) { |file| Document.new(file) }
```

That's more readable, and also safer as, in case there's an exception on the lines of code before open and close, the block syntax will catch the exception and close the file properly before rethrowing, instead of letting the file open.